datasets/counter/counter.py

```diff
@@ -0,0 +1,146 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+import datasets
+
+
+_CITATION = """\
+@Article{Sharjeel2016,
+author="Sharjeel, Muhammad
+and Nawab, Rao Muhammad Adeel
+and Rayson, Paul",
+title="COUNTER: corpus of Urdu news text reuse",
+journal="Language Resources and Evaluation",
+year="2016",
+pages="1--27",
+issn="1574-0218",
+doi="10.1007/s10579-016-9367-2",
+url="http://dx.doi.org/10.1007/s10579-016-9367-2"
+"""
+
+_DESCRIPTION = """\
+ The COrpus of Urdu News TExt Reuse (COUNTER) corpus contains 1200 documents with real examples of text reuse from the field of journalism. It has been manually annotated at document level with three levels of reuse: wholly derived, partially derived and non derived.
+"""
+
+_HOMEPAGE = "http://ucrel.lancs.ac.uk/textreuse/counter.php"
+
+_LICENSE = (
+    "The corpus is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. "
+)
+
+_DOWNLOAD_URL = "http://ucrel.lancs.ac.uk/textreuse/COUNTER.zip"
+
+_NUM_EXAMPLES = 600
+
+
+class Counter(datasets.GeneratorBasedBuilder):
+    """Corpus of Urdu News Text Reuse"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "source": {
+                    "filename": datasets.Value("string"),
+                    "headline": datasets.Value("string"),
+                    "body": datasets.Value("string"),
+                    "total_number_of_words": datasets.Value("int64"),
+                    "total_number_of_sentences": datasets.Value("int64"),
+                    "number_of_words_with_swr": datasets.Value("int64"),
+                    "newspaper": datasets.Value("string"),
+                    "newsdate": datasets.Value("string"),
+                    "domain": datasets.Value("string"),
+                    "classification": datasets.ClassLabel(
+                        num_classes=3, names=["WD", "PD", "ND"], names_file=None, id=None
```

Comment:
Here you can simply use `names`:
`ClassLabel(names=["wholly_derived", "partially_derived", "not_derived"]),`